plugins/ua_network_pubsub_mqtt.h

```diff
@@ -25,12 +25,12 @@ typedef struct {
     UA_Connection *connection;
     void * mqttClient;
     void (*callback)(UA_ByteString *encodedBuffer, UA_ByteString *topic);
+    UA_String *mqttUsername;
```

Comment:
Make these non-pointer UA_String.
And copy the content over.
So we don't keep a dependency on the connectionConfig lifecycle.

Reply:
Thanks for the review!

I hope I have covered all places where the two strings have to be cleared.